exporter/SynthesisFusionGltfExporter/gltf/utils/FusionUtils.py

```diff
@@ -65,23 +65,25 @@ def getPBRSettingsFromAppearance(fusionAppearance: adsk.core.Appearance, exportW
 
     transparent = False
     emissiveColorFactor = None
-    metallicFactor = 0.0
-    roughnessProp = props.itemById("surface_roughness")
-    roughnessFactor = roughnessProp.value if roughnessProp is not None else 0.1
+    metallicFactor = 0.1 # never 0 and certainly never 1
+    roughnessFactor = 0.1
 
+    roughnessProp = props.itemById("surface_roughness")
+    roughnessFactor = roughnessProp.value
     baseColor = None
 
     modelItem = props.itemById("interior_model")
     if modelItem is None:
         return None
     matModelType = modelItem.value
 
+
     if matModelType == 0:  # Opaque
         baseColor = props.itemById("opaque_albedo").value
         if props.itemById("opaque_emission").value:
             emissiveColorFactor = fusionColorToRGBAArray(props.itemById("opaque_luminance_modifier").value)[:3]
-    elif matModelType == 1:  # Metal
-        metallicFactor = 1.0
+    elif matModelType == 1:  # Metal - even metal should be set at most to about .2-.5
+        metallicFactor = 0.3
```

Comment:
#### The Actual Change

Metallic property should really never be 1 unless this is a special type of mirror with unique shaders.

Defaulted to 0.3 because it looked good.